apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/RetryTimings.cs

```diff
@@ -0,0 +1,116 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at 
+//
+// https://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+using System;
+
+namespace Google.Cloud.Storage.V1;
+
+/// <summary>
+/// Custom timing configurations for retrying
+/// </summary>
+public sealed class RetryTimings
+{
+    /// <summary>
+    /// Default initial backoff time
+    /// </summary>
+    public static TimeSpan DefaultInitialBackoff { get; } = TimeSpan.FromSeconds(1);
+
+    /// <summary>
+    /// Default maximum backoff time
+    /// </summary>
+    public static TimeSpan DefaultMaxBackoff { get; } = TimeSpan.FromSeconds(32);
+
+    /// <summary>
+    /// Default retry timing configurations
+    /// </summary>
+    public static RetryTimings Default { get; } = new RetryTimings().WithInitialBackoff(RetryTimings.DefaultInitialBackoff).WithMaxBackoff(RetryTimings.DefaultMaxBackoff);
```

Comment:
What about backoff multiplier? And can't we just provide default parameter values to the constructor?